content/images/logos/windows.svg

```diff
@@ -1,29 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!DOCTYPE svg PUBLIC "-//W3C/DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg" 
    version="1.1"
-   width="220"
-   height="220"
-   id="svg5358">
-  <defs
-     id="defs5360" />
- 
-  <path
-     d="M 104.8675,104.8675 0,104.8675 0,0 l 104.8675,0 0,104.8675 z"
-     id="path5058"
-     style="fill:#f1511b;fill-opacity:1;fill-rule:nonzero;stroke:none" />
-  <path
-     d="m 220.65375,104.8675 -104.86625,0 0,-104.8675 104.86625,0 0,104.8675 z"
-     id="path5060"
-     style="fill:#80cc28;fill-opacity:1;fill-rule:nonzero;stroke:none" />
-  <path
-     d="m 104.865,220.695 -104.865,0 0,-104.8675 104.865,0 0,104.8675 z"
-     id="path5062"
-     style="fill:#00adef;fill-opacity:1;fill-rule:nonzero;stroke:none" />
-  <path
-     d="m 220.65375,220.695 -104.86625,0 0,-104.8675 104.86625,0 0,104.8675 z"
-     id="path5064"
-     style="fill:#fbbc09;fill-opacity:1;fill-rule:nonzero;stroke:none" />
-</svg>
+   height="88"
+   width="88">
+   
+   <path
+      fill="#00adef"
+      d="m0,12.402,35.687-4.8602,0.0156,34.423-35.67,0.20313zm35.67,33.529,0.0277,34.453-35
+      67-4.9041-0.002-29.78zm4.3261-39.025,47.318-6.906,0,41.527-47.318,0.37565zm47.329,39.
+      49-0.0111,41.34-47.318-6.6784-0.0663-34.739z" />
+</svg
```

Comment:
Woops I missed that, the closing tag must end with a `>`, otherwise the image won’t be displayable: https://xmpp.org/images/logos/windows.svg